### ISSUE TITLE
DDA-LUA-Items error on startup

### ISSUE THREAD

**Issue by Szara-ManOfHonor** [ZhilkinSerg/Cataclysm-DDA-LUA#5]
When loading (even with just the LUA mods) the following error occurs

`ERROR : src/item_factory.cpp:2555 [use_function Item_factory::usage_from_string(const string&) const] Received unrecognized iuse function IUSE_DDA_LUA_ENTITY_SCANNER, using iuse::none instead`

Edit:
Well, might be because LUA got axed a few weeks ago. Goddamnit Kevin.
Is there a future for the content of the DDA Lua mods?

**Comment by ZhilkinSerg** [comment]
Yeah, errors are because Lua was axed. Mod would still be working in any version where Lua is still available.

**Comment by ChainsawGarden** [comment]
> 

Fork happening soon.
